easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.0.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Tarball'
+
+name = 'SqueezeMeta'
+version = '1.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/jtamames/SqueezeMeta'
+description = """SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,
+ covering all steps of the analysis."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/jtamames/SqueezeMeta/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['SqueezeMeta-%(version)s_fix-hardcoding.patch']
+checksums = [
+    '35fa45246fa3580e7ab00644176f719151465aa0ed5f0023d30075668baefb11',  # v1.0.0.tar.gz
+    '5fe6b721669d022a1d891472d64061f217791ff8efd1642d34a03de18a21d677',  # SqueezeMeta-1.0.0_fix-hardcoding.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.0'),  # provides XML::Parser
+    ('XML-LibXML', '2.0132', '-Perl-%(perlver)s'),
+    ('Python', '2.7.15'),
+    ('matplotlib', '2.2.3', versionsuffix),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('R', '3.5.1', versionsuffix),
+    ('BLAST+', '2.7.1'),
+    ('BWA', '0.7.17'),
+    #('barrnap', '0.9'),
```

Comment:
@boegel could you appease the style checker here, please?

Reply:
fixed in aebd17a